wave_vision/include/wave/vision/fast_detector.hpp

```diff
@@ -94,9 +98,9 @@ class FASTDetector : public FeatureDetector {
     *  @param check_config, the FASTParams struct containing the desired
     *  configuration values.
     */
-    void checkConfiguration(FASTParams& check_config);
+    void checkConfiguration(const FASTParams& check_config);
 };
 
-} // end of namespace wave
+}  // end of namespace wave
```

Comment:
ah right, I'm not even using cpplint yet. Do you have a config file for that? by default cpplint adheres to Google's style guide no?